<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="editGroupComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="editGroupPartyComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>120</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="editGroupCancelPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Peruuta</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="editGroupSavePushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tallenna</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editGroupNameLineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>113</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Ryhm채</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>47</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Seurue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>70</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Ryhm채n nimi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editGroupPopulatePushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>T채yt채</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editGroupComboBox</tabstop>
  <tabstop>editGroupPopulatePushButton</tabstop>
  <tabstop>editGroupPartyComboBox</tabstop>
  <tabstop>editGroupNameLineEdit</tabstop>
  <tabstop>editGroupCancelPushButton</tabstop>
  <tabstop>editGroupSavePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
